extensions/agent/src/data/pickScheduleData.ts

```diff
@@ -22,10 +22,12 @@ export class PickScheduleData implements IAgentDialogData {
 
 	public async initialize() {
 		let agentService = await AgentUtils.getAgentService();
-		let result = await agentService.getJobSchedules(this.ownerUri);
-		if (result && result.success) {
-			this.schedules = result.schedules;
-		}
+		await agentService.getJobSchedules(this.ownerUri).then((result) => {
+			if (result && result.success) {
```

Comment:
If the result isn't a success we should probably be displaying an error to the user

Reply:
Fixed